engines/efh/metaengine.cpp

```diff
@@ -224,97 +224,470 @@ bool EfhMetaEngine::removeSaveState(const char *target, int slot) const {
+	act->setCustomEngineActionEvent(kActionExitStatusMenu);
+	act->addDefaultInputMapping("ESCAPE");
+	act->addDefaultInputMapping("l");
+	act->addDefaultInputMapping("JOY_B");
+	statusMenuKeymap->addAction(act);
+
+	for (const EfhSubMenuOption *i = _efhSubMenuOptions; i->_id; ++i) {
+		act = new Action(i->_id, _(i->_desc));
+		act->setCustomEngineActionEvent(i->_action);
+		act->addDefaultInputMapping(i->_inputKey);
+		statusMenuSubMenuKeymap->addAction(act);
+	}
+
+	act = new Action("EXIT", _("Exit submenu"));
+	act->setCustomEngineActionEvent(kActionExitSubMenu);
+	act->addDefaultInputMapping("ESCAPE");
+	act->addDefaultInputMapping("JOY_B");
+	statusMenuSubMenuKeymap->addAction(act);
+
+	act = new Action("STARTFIGHT", _("Attack"));
+	act->setCustomEngineActionEvent(kActionStartFight);
+	act->addDefaultInputMapping("a");
+	act->addDefaultInputMapping("JOY_UP");
+	interactionKeymap->addAction(act);
+
+	act = new Action("LEAVE", _("Leave"));
+	act->setCustomEngineActionEvent(kActionLeave);
+	act->addDefaultInputMapping("l");
+	act->addDefaultInputMapping("ESCAPE");
+	act->addDefaultInputMapping("JOY_DOWN");
+	interactionKeymap->addAction(act);
+
+	act = new Action("STATUS", _("Status"));
+	act->setCustomEngineActionEvent(kActionStatus);
+	act->addDefaultInputMapping("s");
+	act->addDefaultInputMapping("JOY_LEFT");
+	interactionKeymap->addAction(act);
+
+	act = new Action("TALK", _("Talk"));
+	act->setCustomEngineActionEvent(kActionTalk);
+	act->addDefaultInputMapping("t");
+	act->addDefaultInputMapping("JOY_RIGHT");
+	interactionKeymap->addAction(act);
+
+	act = new Action("ATTACK", _("Attack"));
+	act->setCustomEngineActionEvent(kActionAttack);
+	act->addDefaultInputMapping("a");
+	act->addDefaultInputMapping("JOY_UP");
+	fightKeymap->addAction(act);
+
+	act = new Action("DEFEND", _("Defend"));
+	act->setCustomEngineActionEvent(kActionDefend);
+	act->addDefaultInputMapping("d");
+	act->addDefaultInputMapping("JOY_DOWN");
+	fightKeymap->addAction(act);
+
+	act = new Action("HIDE", _("Hide"));
+	act->setCustomEngineActionEvent(kActionHide);
+	act->addDefaultInputMapping("h");
+	act->addDefaultInputMapping("JOY_LEFT");
+	fightKeymap->addAction(act);
+
+	act = new Action("RUN", _("Run"));
+	act->setCustomEngineActionEvent(kActionRun);
+	act->addDefaultInputMapping("r");
+	act->addDefaultInputMapping("JOY_RIGHT");
+	fightKeymap->addAction(act);
+
+	act = new Action("STATUS", _("Status"));
+	act->setCustomEngineActionEvent(kActionTeamStatus);
+	act->addDefaultInputMapping("s");
+	act->addDefaultInputMapping("JOY_Y");
+	fightKeymap->addAction(act);
+
+	// I18N: This action is used to view the terrain during a fight so that the player can see where they can move if they decide to run.
+	act = new Action("TERRAIN", _("View terrain"));
+	act->setCustomEngineActionEvent(kActionTerrain);
+	act->addDefaultInputMapping("t");
+	act->addDefaultInputMapping("JOY_X");
+	fightKeymap->addAction(act);
+
+	act = new Action("CHARACTER1", _("Select character 1"));
```

Comment:
```suggestion
	// Character is a hero in-game
	act = new Action("CHARACTER1", _("Select character 1"));
```

E.g. it is not a symbol. And add similar message to the translators in other places, just copy/paste